core/README.md

```diff
@@ -65,6 +65,8 @@ Infrastructure interface includes:
   must be a singleton.
 * [Uploader](./module/README.md): Uploader is the modular to handle put object request from user 
   account, and store it in primary SP's piece store.
+* [Metadata](./module/README.md): Metadata is the modular to handle the get request of metadata of objects,
```

Comment:
Metadata module have not define the interface in core level, it is can not offer the customization ability.
This is not one of the standardized processes in the gfsp framework, so it is not recommended to add the introduction in the core layer. This will cause misleading, you can add it in modular/metadata/README.md

Reply:
done